layouts/partials/jslibraries.html

```diff
@@ -183,3 +183,41 @@
 
   amplitude.getInstance().logEvent("Page loaded");
 </script>
+
```

Comment:
This is not a library. Do not keep it in jslibraries.html. Include it in some other file and then include it on the top.